www/script.js

```diff
@@ -0,0 +1,186 @@
+mapping = {};
+mapping["sinkhole"] = "c1";
```

Comment:
we're going to have to remember to update this every time the layout of the lab changes, which is pretty much at least once a semester when we do the big cleanup. is there a way we can possibly make this easier outside of documenting that it must be done?

Reply:
I think last time we were able to put the desktops back in the same places? Even so, I think the best place for that kind of info would be in ocf-etc.